client/src/components/forms/CommentEditor.vue

```diff
@@ -112,14 +114,26 @@ const props = defineProps({
     type: Object,
     default: () => {},
   },
+  comment: {
+    type: Object,
+    required: false,
+    default: null,
+  },
 })
 
+const defaultContent = computed(() => {
+  if (!props.parent?.id) return ""
+  if (props.parent.id === props.replyTo?.id) return ""
+  //TODO: Make this more robust to handle multi paragraphs, etc
+  return `<blockquote>${props.replyTo.content}</blockquote>`
```

Comment:
```suggestion
  return `<blockquote>${props.replyTo.content}</blockquote><p></p>`
```
Minimizing the chance that the user accidentally modifies the content of the blockquote and preventing them from having to hit the return key twice is preferable in my opinion.

Reply:
Nice.  I like it.